doc/api-extensions.md

```diff
@@ -2328,3 +2328,9 @@ This is indicated when command `lxc version` is executed or when `/1.0` endpoint
 This API extension enables setting an `oidc.groups.claim` configuration key.
 If OIDC authentication is configured and this claim is set, LXD will request this claim in the scope of OIDC flow.
 The value of the claim will be extracted and might be used to make authorization decisions.
+
+## `auto_stateful_migration`
+
+This API extension allows newly created VMs to have their `migration.stateful` configuration key automatically set to `true`
+through the new server-level configuration key `instances.migration.stateful`. If `migration.stateful` is already manually set to
+`true` at the instance level or at the profile level, `instances.migration.stateful` is not applied.
```

Comment:
Should be:

> If `migration.stateful` is already set at the profile or instance level then `instances.migration.stateful` is not applied.

No need to mention `true` here, as equally it could be explicitly set to `false` the lower levels.